packages/async-operations/tests/test-async_operations.js

```diff
@@ -0,0 +1,359 @@
+'use strict';
+
+const cryptoRandomString = require('crypto-random-string');
+
+const test = require('ava');
+const sinon = require('sinon');
+
+const omit = require('lodash/omit');
+const { ecs, lambda, s3 } = require('@cumulus/aws-client/services');
+const { recursivelyDeleteS3Bucket } = require('@cumulus/aws-client/S3');
+// eslint-disable-next-line node/no-unpublished-require
+const { randomString } = require('@cumulus/common/test-utils');
+const {
+  localStackConnectionEnv, createTestDatabase, deleteTestDatabase, getKnexClient,
+} = require('@cumulus/db');
+const { EcsStartTaskError } = require('@cumulus/errors');
+
+// eslint-disable-next-line node/no-unpublished-require
+const { migrationDir } = require('../../../lambdas/db-migration');
+
+const { getLambdaEnvironmentVariables, startAsyncOperation } = require('../dist/async_operations');
+
+const dynamoTableName = 'randomDynamoTable';
```

Comment:
Is this necessary or does it need to be truly random?

Reply:
No - it could be named something better. 

How about: 4e61ff4f9